recipe/meta.yaml

```diff
@@ -2,20 +2,23 @@
 # See the guidelines for updating this recipe for new releases:
 # https://github.com/scrapy/scrapy/wiki/Scrapy-release-procedure#conda-packages
 #
-{% set name = "Scrapy" %}
-{% set version = "2.11.1" %}
+{% set name = "scrapy" %}
+{% set version = "2.12.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 733a039c7423e52b69bf2810b5332093d4e42a848460359c07b02ecff8f73ebe
+  sha256: d66d6e76009b12447604196875a463b61d10721140032a8084a0a52df7f4788f
+  patches:  # [win]
+    # This should be able to be removed once libxml2 is updated. 
```

Comment:
Updated to which version? Can the comment be updated so that the next updated knows when to remove the patch?

Reply:
I only tested the latest version on conda forge. There might be an earlier version that would work, do you want me to try and find the exact version?

Reply:
We have libxml2 2.13.1 and the latest I see is 2.13.5 (https://gitlab.gnome.org/GNOME/libxml2/-/releases)
Based on our current run_exports, a rebuild of downstreams wouldn't be necessary. If 2.13.5 fixes this issue, we should build it instead of patching here.

Reply:
Ok that's good to know, I thought that would be a massive rebuild. We'll give this a shot.

Reply:
Building with the latest libxml2/lxml works without the hacks. 👍